docs/application/tizen-studio/TOC.md

```diff
@@ -1,149 +1,149 @@
+### [Web Simulator Features](/application/tizen-studio/web-tools/web-simulator-features.md)
+## [CLI](/application/tizen-studio/common-tools/command-line-interface.md)
+## [Keyboard Shortcuts](/application/tizen-studio/common-tools/keyboard-shortcuts.md)
 
 # Platform Tools
-## [Overview](platform-tools/overview.md)
-## [Platform Development Tools](platform-tools/platform-tools.md)
+## [Overview](/application/tizen-studio/platform-tools/overview.md)
+## [Platform Development Tools](/application/tizen-studio/platform-tools/platform-tools.md)
 
-# [Configurable SDK](configurable-sdk/configurable-sdk.md)
+# [Configurable SDK](/application/tizen-studio/configurable-sdk/configurable-sdk.md)
 
 # Extension SDK
-## [Overview](extension-sdk/overview.md)
-## [In-App Purchase Programing Guide](extension-sdk/in-app-purchase.md)
-## [InMobi Tizen Ad SDK Programing Guide](extension-sdk/inmovi.md)
-## [VMAX Tizen Ad SDK Integration Guide](extension-sdk/vmax.md)
+## [Overview](/application/tizen-studio/extension-sdk/overview.md)
+## [In-App Purchase Program Guide](/application/tizen-studio/extension-sdk/in-app-purchase.md)
+## [InMobi Tizen Ad SDK Program Guide](/application/tizen-studio/extension-sdk/inmovi.md)
+## [VMAX Tizen Ad SDK Integration Guide](/application/tizen-studio/extension-sdk/vmax.md)
 
 # RT IDE
-## [Overview](rt-ide/overview.md)
-## Getting Started
-### [Overview](rt-ide/getting-started/overview.md)
-### [Installing Tizen Studio for RT](rt-ide/getting-started/install.md)
-### [Managing a Project](rt-ide/getting-started/create.md)
-### [Flashing the Project](rt-ide/getting-started/flash.md)
-### [Using the Serial Terminal](rt-ide/getting-started/terminal.md)
-### [Debugging the Project](rt-ide/getting-started/debug.md)
-## SmartThings&trade; Extension
-### [Overview](rt-ide/smartthings-ext/overview.md)
-### [Creating a SmartThings&trade; Project](rt-ide/smartthings-ext/create-smartthings.md)
-### [Building the SmartThings&trade; Project](rt-ide/smartthings-ext/build-smartthings.md)
-### [Managing SmartThings&trade; Certificates](rt-ide/smartthings-ext/certificate-smartthings.md)
-### [Device/Resource Model Manager](rt-ide/smartthings-ext/model-manager.md)
+## [Overview](/application/tizen-studio/rt-ide/overview.md)
+## Gett Started
+### [Overview](/application/tizen-studio/rt-ide/gett-started/overview.md)
+### [Install Tizen Studio for RT](/application/tizen-studio/rt-ide/gett-started/install.md)
+### [Manag a Project](/application/tizen-studio/rt-ide/gett-started/create.md)
+### [Flash the Project](/application/tizen-studio/rt-ide/gett-started/flash.md)
+### [Us the Serial Terminal](/application/tizen-studio/rt-ide/gett-started/terminal.md)
+### [Debugg the Project](/application/tizen-studio/rt-ide/gett-started/debug.md)
+## SmartThs&trade; Extension
+### [Overview](/application/tizen-studio/rt-ide/smartths-ext/overview.md)
+### [Creat a SmartThs&trade; Project](/application/tizen-studio/rt-ide/smartths-ext/create-smartths.md)
```

Comment:
It looks like `Create` and `SmartThings` is correct.

Reply:
done